go.mod

```diff
@@ -1,6 +1,6 @@
 module go.etcd.io/etcd/v3
 
-go 1.21
+go 1.21.5
```

Comment:
I think there's a version file for this now, looking into the previous bump:
https://github.com/etcd-io/etcd/commit/cc130fd2d4b36737d392e12b7d24f951f644c1e6#diff-69c69535f549f1e70ce037415e46a2894f04d4147c7929de6ef1db8f72a0a136